EIPS/eip-875.md

```diff
@@ -74,9 +86,7 @@ contract ERC875
   function transferFrom(address _from, address _to, uint256[] _tokens) public;
 
   //optional
```

Comment:
There is no concept of optional functions. You'd need to specify another interface.